Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -60,6 +60,7 @@
     'AP_ICEngine',
     'AP_Frsky_Telem',
     'AP_FlashStorage',
+    'AP_UAVCAN',
```

Comment:
I can state the same for 50% of other libraries... But as the overall tendency goes, it should be included all the time as support is growing fast.